_pages/redirects/community/get-involved-contact.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Old Get involved page redirect
```

Comment:
@kierisi when we "deprecate" a page we use a redirect. it's not perfect but because we are hosted now on github pages we can't do server side redirects at this point. (which are much better for SEO). So we just create a manual redirect via the static site which works for now. 

notice i have a directory fpr redirects in the _pages dir :) 
then you just make sure the page has the original link and then a redirect_to that tells it where to send the user. that redirect to ends up i think as an html refresh element in the meta tags at the top. 

hacky i know but it works good enough for now.